uber/templates/signups/login.html

```diff
@@ -19,15 +19,15 @@ <h2 class="form-signin-heading">Volunteer Login</h2>
         </form>
     {% else %}
         <div class="well">
-            Shifts are not yet available. If you are signed up as a volunteer, you will 
+            Shifts are not yet available. If you are signed up as a volunteer, you will
```

Comment:
Please check to make sure deleting this space won't result in text that renders as `you willbe emailed`.